main.js

```diff
@@ -427,9 +460,16 @@ function logRide() {
 
 
 function reset() {
-
     frogX = 200;
     frogY = 530;
+    //checks if all home spaces are occupied
+    if (JSON.stringify(homeSpaceArray) == JSON.stringify(victoryCheckArray)){
```

Comment:
Instead you can use something like indexOf on your homeSpaceArray checking to see if there are any 0's in it. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf
Something like:

if(homeSpaceArray.indexOf(0) == -1) {}

That way you don't have to have two arrays and you don't have to use JSON.stringify

Reply:
..yeah..that makes way more sense